test/tests/time_steppers/logconstant_dt/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [./logconstant_dt]
+    type = 'Exodiff'
+    input = 'logconstant_dt.i'
+    exodiff = 'logconstant_dt_out.e'
```

Comment:
If you want to be thrifty then you make this a csv test. Afterall, all you need to test is that you are hitting the same times.